File: eSalon/eSalon.Model/VrstaUsluge.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eSalon.Model
{
    public class VrstaUsluge
    {
        public int VrstaId { get; set; }

        public string Naziv { get; set; } = null!;

        public byte[]? Slika { get; set; }
    }
}

File: eSalon/UI/esalon_desktop/lib/models/vrsta_usluge.dart
import 'package:json_annotation/json_annotation.dart';

part 'vrsta_usluge.g.dart';

@JsonSerializable()
class VrstaUsluge {
  int? vrstaId;
  String? naziv;
  String? slika;

  VrstaUsluge({
    this.vrstaId,
    this.naziv,
    this.slika,
  });

  factory VrstaUsluge.fromJson(Map<String, dynamic> json) =>
      _$VrstaUslugeFromJson(json);

  /// Connect the generated [_$PersonToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$VrstaUslugeToJson(this);

}